templates/ui-frameworks/svelte/entry-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{pascal_case entry_type.name}}Detail.svelte.hbs

```diff
@@ -95,11 +104,11 @@ onMount(async () => {
   original{{pascal_case entry_type.name}}Hash={ {{camel_case entry_type.name}}Hash}
   {{/if}}
   currentRecord={record}
```

Comment:
I see this is not caused by this PR, but I'm getting a ts warning here: `Type 'Record | undefined' is not assignable to type 'Record'`

Reply:
Yes that fixes it -- my mistake it appeared after i updated @sveltejs/vite-plugin-svelte, @tsconfig/svelte to 5.x and vite to 6.x. We probably want to update those with this PR.

Reply:
done - https://github.com/holochain/scaffolding/pull/486/commits/f0c1146a9bb03de6cb01ded3aa15193e8f3588d5